README.md

```diff
@@ -180,12 +180,10 @@ healthy
 
 Some orchestration systems, such as Portainer, don't allow for disabling the default `HEALTHCHECK` declared by this image. In those cases you can approximate the disabling of healthchecks by setting the environment variable `DISABLE_HEALTHCHECK` to `true`.
 
-## Autopause (experimental)
+## Autopause
```

Comment:
Are you totally comfortable removing the "experimental" designation? I personally don't have the experience with it to support it, so I'll continue to delegate to you if you don't mind.

Reply:
From what I gathered, after resolving these interface problems and the multiarch tag, this should work now. I'm happy to provide support if needed. We could retain the experimental tag, because this is in no way supported / intended by Mojang if you want.